### DISCUSSION TITLE
Where do I find the os.log for this app?

### DISCUSSION THREAD

**Discussion by shirehobbits** [MonitorControl/MonitorControl#373] [Q&A] [answered]
My brightness control is disabled, need help in debugging.

<img width="197" alt="Screen Shot 2021-01-14 at 11 04 16 AM" src="https://user-images.githubusercontent.com/49618139/104636460-3f0f2500-5658-11eb-9589-3426ca117772.png">

**Comment by JoniVR** [comment, marked as answer]
I think usually inside Console.app, though most of our os.log calls are ifdeffed out so you won't see them.
